config/routes.rb

```diff
@@ -9,7 +9,7 @@
 
   namespace :api do
     namespace :v1 do
-      resources :todos, only: [:destroy]
+      resources :todos, only: [:create, :destroy]
```

Comment:
Well done whitelisting only those actions that are needed and exist, nice work :+1: